pyboy/utils.py

```diff
@@ -151,7 +151,11 @@ class WindowEvent:
         _INTERNAL_RENDERER_FLUSH,
         _INTERNAL_MOUSE,
         _INTERNAL_MARK_TILE,
-    ) = range(36)
+        PRESS_BUTTON_J,
+        PRESS_BUTTON_K,
+        PRESS_BUTTON_LSHIFT,
+        PRESS_BUTTON_RSHIFT
```

Comment:
The events should represent the action they are performing, not the button

Reply:
Okay, i can change this.